### DISCUSSION TITLE
Show windows of current workspace?

### DISCUSSION THREAD

**Discussion by blueray453** [davatorium/rofi#1868] [Q&A] [answered]
I can show windows using `rofi -steal-focus -show window`.

How can I show window of current workspace?

I was playing with it. `xdotool search --all --onlyvisible --desktop 0 ""` gives all windows of workspace 0. 

```
$  xdotool search --all --onlyvisible --desktop 0 ""
Defaulting to search window name, class, and classname
102760455
41943043
100663358
6291466
6303328
6306999
67108868
```
However, I am interested with the looks of `rofi -steal-focus -show window` (I mean the looks and everything). How can I achieve that?

**Comment by yusufaktepe** [comment, marked as answer]
Use `windowcd` mode instead of `window`.

```sh
rofi -steal-focus -show windowcd
```

**Reply by blueray453** [reply]
Thank you very much.

**Reply by tracure1337** [reply]
Love it!
